packages/js/tsconfig.json

```diff
@@ -1,22 +1,30 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "lib": ["ESNext"],
+    "strict": true,
     "target": "ESNext",
     "module": "ESNext",
+    "lib": ["DOM", "DOM.Iterable", "ESNext"],
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
```

Comment:
☑ **chore:** seems like these are not needed:
`moduleResolution` - default value is `node` when `module` is `ESNext`
`resolveJsonModule` - no need for json imports for now
`esModuleInterop and isolatedModules` - not sure about, but it works without it